src/test/java/microsoft/exchange/webservices/data/core/EwsUtilitiesTest.java

```diff
@@ -54,6 +56,11 @@ public void testGetBuildVersion() {
   }
 
   @Test
+  public void testParseInt() throws IllegalAccessException, ParseException, InstantiationException {
+    Assert.assertEquals(new Integer(14), EwsUtilities.parse(Integer.class, "14"));
+  }
```

Comment:
maybe you can also add tests for: `Boolean`, `Double`, `Date`... (including `Number` types). I think we should also extend the bounds (min,max e.g.).